### ISSUE TITLE
[Feature Request] Add v7 to FFmpeg Version setting list

### ISSUE THREAD

**Issue by Korb** [NickeManarin/ScreenToGif#1315]
Currently, FFmpeg version `7`.0.1 (2024-05-26) is [available for download on gyan.dev](https://www.gyan.dev/ffmpeg/builds/#release-builds). However, ScreenToGif on the Extras tab of the Options window only offers options `v4`, `v5` and `v6`.

**Comment by panther7** [comment]
Yes, need supports for FFMpeg v7, because some options is deprecated/broken in v7 and encoding crashing...
